start.sh

```diff
@@ -15,6 +15,7 @@ if [ "$1" != "sha" ]; then
     fi
 
     # Start services.
+    [ ! -d /run/sendsigs.omit.d ] && mkdir /run/sendsigs.omit.d
```

Comment:
how is this different than just `mkdir -p`?

Reply:
i forgot about the -p option. this line still needs more investigation. it keeps an unexpected line from appearing in the log file, but the server is acting weird now.